package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "lint": "eslint .",
-    "lint-diff": "git diff --name-only --cached --relative --diff-filter=ACM | grep \\.js$ | xargs eslint",
+    "lint-diff": "git diff --name-only --cached --relative --diff-filter=ACM | grep \\\\.js$ | xargs eslint",
```

Comment:
Why this change?

Reply:
this line threw me an error 'cause was not escaping the backslashes